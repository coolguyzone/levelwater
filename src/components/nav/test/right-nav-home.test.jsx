import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RightNavHome from '../right-nav-home.jsx';


describe('right nav home', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<RightNavHome />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
