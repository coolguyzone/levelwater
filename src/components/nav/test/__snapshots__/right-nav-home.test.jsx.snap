exports[`right nav home renders a snapshot 1`] = `
<div
  className="ui-buttons">
  <Link
    replace={false}
    to="/login">
    <button
      className="ui button">
      Log In
    </button>
  </Link>
  <Link
    replace={false}
    to="/signup">
    <button
      className="ui button">
      Sign Up
    </button>
  </Link>
</div>
`;
